src/components/CompletedTask/CompletedTask.tsx

```diff
@@ -35,11 +35,13 @@ const CompletedTask = ({
     setExpanded(!expanded);
   };
 
+const showArrow = (description || image ) 
```

Comment:
How the mismatched spacing makes me feel
![Alt Text](https://giphy.com/embed/ZKZiW6GSx8eSA)